Src/FluentAssertions/Primitives/StringAssertions.cs

```diff
@@ -2007,6 +2011,20 @@ private static string ApplyStringSettings(string value, IEquivalencyOptions opti
             value = value.TrimEnd();
         }
 
+        if (options.IgnoreAllNewlines)
+        {
+            value = value
+                .Replace("\r", string.Empty, StringComparison.Ordinal)
+                .Replace("\n", string.Empty, StringComparison.Ordinal);
+        }
+
+        if (options.IgnoreNewlineStyle)
+        {
+            value = value
+                .Replace("\r\n", "\n", StringComparison.Ordinal)
```

Comment:
🤔 Can't you use string.ReplaceLineEndingsfor this?

Reply:
[`ReplaceLineEndings`](https://learn.microsoft.com/en-us/dotnet/api/system.string.replacelineendings) is only supported in .NET 6 or newer.
I could polyfill it, but think that this would be better implemented in a separate refactoring, as we also have other locations in the library where we work with newlines and should be consistent.

Note:
`ReplaceLineEndings` also replaces other character sequences that are treated as newlines:
> The list of recognized newline sequences is CR (U+000D), LF (U+000A), CRLF (U+000D U+000A), NEL (U+0085), LS (U+2028), FF (U+000C), and PS (U+2029). This list is given by the Unicode Standard, Sec. 5.8, Recommendation R4 and Table 5-2.

Reply:
Fair enough